miniapp/miniapp.cpp

```diff
@@ -101,15 +107,13 @@ int main(int argc, char** argv) {
             report_compartment_stats(*recipe);
         }
 
-        // inject some artificial spikes, 1 per 20 neurons.
-        std::vector<cell_gid_type> local_sources;
+        // Inject some artificial spikes, 1 per 20 neurons.
         cell_gid_type first_spike_cell = 20*((cell_range.first+19)/20);
         for (auto c=first_spike_cell; c<cell_range.second; c+=20) {
-            local_sources.push_back(c);
```

Comment:
To get reliable results from the metering the "dummy" step of the model to initialize the profiler had to be removed. This simplified the code a bit, and it was no longer necessary to record the seed spikes in a `local_sources` vector for re-insertion after the dummy step.